<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="LAPTOP-9KKEIR94" ignored="0" name="Testcasepractice" tests="3" failures="1" timestamp="19 Nov 2021 19:29:11 GMT" time="21.912" errors="0">
  <testcase name="validUserShouldBeAbleToAddCategory" time="10.677" classname="test.AddCategoryTest"/>
  <testcase name="validateAllMonthsPresentInTheDropDown" time="4.671" classname="test.AddCategoryTest"/>
  <testcase name="validatingOnlyUniqueOrdersCanBeAdded" time="6.535" classname="test.AddCategoryTest">
    <failure type="java.lang.AssertionError" message="Duplicate category not allowed expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: Duplicate category not allowed expected [true] but found [false]
at test.AddCategoryTest.validatingOnlyUniqueOrdersCanBeAdded(AddCategoryTest.java:47)
... Removed 30 stack frames]]>
    </failure>
  </testcase> <!-- validatingOnlyUniqueOrdersCanBeAdded -->
</testsuite> <!-- Testcasepractice -->
